lib/elixir/lib/macro.ex

```diff
@@ -2515,19 +2515,6 @@ defmodule Macro do
   end
 
   defp dbg_format_ast(ast) do
-    [:bright, ast, :reset, :faint, " #=>", :reset]
-  end
-
-  defp dbg_default_syntax_colors do
-    [
-      atom: :cyan,
-      string: :green,
-      list: :default_color,
-      boolean: :magenta,
-      nil: :magenta,
-      tuple: :default_color,
-      binary: :default_color,
-      map: :default_color
-    ]
+    [ast, :faint, " #=>", :reset]
```

Comment:
@whatyouhide I removed the bright around the ast because it was calling too much attention to it, while I think we should focus on the result. :)

Reply:
@josevalim I think this makes hard to distinguish AST and result. Maybe we should make the result bright instead?